src/Collectors/EventDataCollector.php

```diff
@@ -29,24 +22,16 @@ abstract class TimelineDataCollector implements DataCollector
     /** @var float */
     protected $request_start_time;
 
-    public function __construct(Application $app, Agent $agent)
```

Comment:
The child classes use $app but no longer have it.

Reply:
You are absolutely right. Adding it right now.